src/domain/synthetics/common/useIncentiveStats.ts

```diff
@@ -1,22 +1,16 @@
+import useSWR from "swr";
 import { RawIncentivesStats, useOracleKeeperFetcher } from "../tokens";
-import { useEffect, useState } from "react";
 
 export default function useIncentiveStats(chainId: number) {
```

Comment:
does it make sense to make chainId optional?

Reply:
Let's do. in all other places we provide chainId. But it might make sense to have overriding arg instead everywhere at some point